src/components/modalPrompt/ModalPrompt.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Modal } from '../modal/Modal';
+import { ModalBody } from '../modal/ModalBody';
+import { ModalFooter } from '../modal/ModalFooter';
+import { ModalBackdrop } from '../modal/ModalBackdrop';
+
+export interface IModalPromptOwnProps {
+  id: string;
+  title: string;
+  message: string;
+  confirmLabel: string;
+  cancelLabel: string;
```

Comment:
You could probably have default value for confirmLabel & cancelLabel and make those properties optional